itests/deals_concurrent_test.go

```diff
@@ -63,7 +63,7 @@ func TestDealWithMarketAndMinerNode(t *testing.T) {
 		ns := fmt.Sprintf("%d", n)
 		t.Run(ns+"-fastretrieval-CAR", func(t *testing.T) { runTest(t, n, true, true) })
 		t.Run(ns+"-fastretrieval-NoCAR", func(t *testing.T) { runTest(t, n, true, false) })
-		t.Run(ns+"-stdretrieval-CAR", func(t *testing.T) { runTest(t, n, true, false) })
+		t.Run(ns+"-stdretrieval-CAR", func(t *testing.T) { runTest(t, n, false, true) })
```

Comment:
@dirkmc Why have we made this change ?

Reply:
This fixes a bug in the test - the name of the test is `stdretrieval-CAR` but the parameters are wrong so the test duplicates `fastretrieval-NoCAR`